src/index.ts

```diff
@@ -435,6 +461,22 @@ export class LoggingBunyan extends Writable {
       (this.cloudLog as Log).write(entries, this.generateCallback(callback));
     }
   }
+
+  /**
+   * Method used to retrieve the current logging-bunyan library version
+   * @returns The version of this library
+   */
+  getNodejsLibraryVersion() {
```

Comment:
Should this be private, or should end users use it?

Reply:
I am fine with this method being used by users if they want - there is absolutely no real reason to hide it